components/library/gd2/patches/000-cmake-soname.patch

```diff
@@ -0,0 +1,170 @@
+--- libgd-gd-2.2.4/CMakeLists.txt.orig	2017-06-10 01:26:44.907495342 +0000
```

Comment:
It comes from their 2.1 branch. For some reason their 2.2 branch is behind when it comes to CMake files.